pkg/storage/types/pvc.go

```diff
@@ -155,3 +157,45 @@ func GetSizeIncludingFSOverhead(size *resource.Quantity, storageClass *string, v
 	cdiFSOverhead := GetFilesystemOverhead(volumeMode, storageClass, cdiConfig)
 	return GetSizeIncludingGivenOverhead(size, cdiFSOverhead)
 }
+
+func GetPersistentVolumeClaimFromCache(namespace, name string, pvcInformer cache.SharedIndexInformer) (*k8sv1.PersistentVolumeClaim, error) {
+	key := controller.NamespacedKey(namespace, name)
+	obj, exists, err := pvcInformer.GetStore().GetByKey(key)
+
+	if err != nil {
+		return nil, fmt.Errorf("error fetching PersistentVolumeClaim %s: %v", key, err)
+	}
+	if !exists {
+		return nil, nil
+	}
+
+	pvc, ok := obj.(*k8sv1.PersistentVolumeClaim)
+	if !ok {
+		return nil, fmt.Errorf("error converting object to PersistentVolumeClaim: object is of type %T", obj)
+	}
+
+	return pvc, nil
+}
+func HasPVCBinding(namespace string, volumes []virtv1.Volume, pvcInformer cache.SharedIndexInformer) bool {
```

Comment:
**Binding** can be used as adjective and noun. And this is misleading. 

I think **HasUnboundPvcs** would be easier to understand.